### PR TITLE
Fix typo in become script

### PR DESCRIPTION
**Pull request by se7entyse7en**
Signed-off-by: Lou Marvin Caraig <loumarvincaraig@gmail.com>

### CODE REVIEW

File: server/athenian/api/become.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def main():
     bearer = sys.argv[1]
     user = sys.argv[2] if len(sys.argv) > 2 else ""
     if not user.startswith("github|") and user:
-        with urlopen("https://api.github.com/users/" + user) as http:
+        with urlopen("https://api.github.com/user/" + user) as http:
```
Comment (vmarkovtsev) [inline server/athenian/api/become.py:14]: I wonder how it worked (and it did work) before...

Comment (vmarkovtsev) [reply server/athenian/api/become.py:14]: Actually, this does not work.

```
https://api.github.com/user/eiso
```

shows an error. This code is needed to resolve users by their logins.

Comment (vmarkovtsev) [reply server/athenian/api/become.py:14]: I think you probably misused the script. It has two modes:
1. `become.py <token> eiso` by gh login
2. `become.py <token> 'github|12345678'` by gh id

**Comment by vmarkovtsev** [comment]
Assuming resolved.

**Comment by se7entyse7en** [comment]
Now I recall the problem 😄. It's because I was passing the id instead of the username and for some reason:
- https://api.github.com/users/se7entyse7en works
- https://api.github.com/user/se7entyse7en doesn't work
- https://api.github.com/users/5599208 doesn't work
- https://api.github.com/user/5599208 works

File: server/athenian/api/become.py (hunk 1 above)
Comment (se7entyse7en) [reply server/athenian/api/become.py:14]: ah indeed just read now 👍

**Pull request closed without merging**